help_to_heat/frontdoor/views.py

```diff
@@ -835,6 +843,14 @@ def build_extra_context(self, request, session_id, page_name, data, is_change_pa
         epc_band = epc.get("current-energy-rating")
         epc_date = epc.get("lodgement-date")
 
+        epc_property_type = epc.get("property-type").upper()
+
+        if epc_property_type in property_types:
+            property_type = property_types[epc_property_type]
+        else:
+            logger.error(f"Unrecognised Property Type: {epc_property_type}")
+            property_type = ""
```

Comment:
could there be a more ergonomic fallback value here? None?

Reply:
I think None is easier to read, yeah. I've made the change for the Date field as well, so both fallback values are consistent.